client/www/pages/dash/index.tsx

```diff
@@ -329,6 +335,24 @@ function Dashboard() {
     });
   }, [token]);
 
+  // TODO: do
+  useEffect(() => {
```

Comment:
We may want to refactor how we create and claim ephemeral apps. I will look into this after the essay.